composer.lock

```diff
@@ -4,15 +4,15 @@
         "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#installing-dependencies",
         "This file is @generated automatically"
     ],
-    "content-hash": "9d46e186da8fa0e68aae4ef50354b998",
+    "content-hash": "d1a85e21f2c175eaef58152407cc6756",
     "packages": [
         {
             "name": "ampproject/common",
-            "version": "dev-develop",
+            "version": "dev-fix/4604-parse-turn-unit",
```

Comment:
You seem to have modified the Composer lock file so that it points to this branch for its libraries. I think that that will break as soon as this is merged, as the `composer install` will not be able to find the branch that is being referenced in the lock file.

Reply:
This might be caused by the recent change I did to avoid locking the libraries to a branch so we can cross the v2 branch barrier...

What exactly did you run? Only a `composer update` in the root folder?

Reply:
> What exactly did you run? Only a composer update in the root folder?

Yep.

Reply:
Copied from Slack:
> Ok, I got a usable lock file now. However, I did it by running `composer update` from `dev-develop` after having deleted the `vendor` folder. Then I changed into the `fix/4604-parse-turn-unit` branch and re-ran `composer update` without  deleting the `vendor` folder.